models/src/routers/training_routers.py

```diff
@@ -223,12 +229,14 @@ def delete_training_by_id(
     if not training:
         LOGGER.info("Delete_training get not existing id %s", request.id)
         raise HTTPException(status_code=404, detail="Not existing training")
+    bucket_name = convert_bucket_name_if_s3prefix(x_current_tenant)
+    training.tenant = convert_bucket_name_if_s3prefix(training.tenant)
```

Comment:
You sure we have to change `tenant` to bucket name? Bucket name is S3 story only, we shouldn't mix it up with tenants

Reply:
You're right. We don't have to use it. I've rewritten this to use bucket_name